base/stats.go

```diff
@@ -61,6 +62,33 @@ const (
 
 const StatsGroupKeySyncGateway = "syncgateway"
 
+const (
+	PrometheusValueTypeGauge   = "gauge"
+	PrometheusValueTypeCounter = "counter"
+
+	StatUnitNoUnits       = ""
+	StatUnitPercent       = "percent"
+	StatUnitBytes         = "bytes"
+	StatUnitNanoseconds   = "nanoseconds"
+	StatUnitSeconds       = "seconds"
+	StatUnitUnixTimestamp = "unix timestamp"
+
+	StatFormatInt      = "int"
+	StatFormatFloat    = "float"
+	StatFormatDuration = "duration"
+	StatFormatBool     = "bool"
+
+	StatAddedVersion3dot0dot0 = "3.0.0"
+	StatAddedVersion3dot1dot0 = "3.1.0"
+	StatAddedVersion3dot2dot0 = "3.2.0"
+
+	StatDeprecatedVersionNotDeprecated = ""
+
+	StatStabilityCommitted = "committed"
+	StatStabilityVolatile  = "volatile"
+	StatStabilityInternal  = "internal"
+)
+
```

Comment:
I think most of these are not needed to be backported. Should I have marked `DatabaseBucketMismatches` as something new in 3.1.2?

Reply:
Oh crap, I think this was a bad conflict resolution. I remember intentionally not pulling them over in the cherry-pick at one point.

Will redo tomorrow.